query.sql

```diff
@@ -0,0 +1,63 @@
+\c workout_tracker_test
+
+
+
+/*
+INSERT INTO users (username, password, email, gender, weight, height) VALUES ('sherlock', 'infiltrate', 'sherlock@example.com', 'male', '124', 6.7);
+INSERT INTO users (username, password, email, gender, weight, height) VALUES ('watson', 'infiltrate', 'watson@example.com', 'male', '80', 5.2);
+*/
+INSERT INTO workouts (workout_name, workout_description, user_id) VALUES ('Triceps, Biceps and Calves', 'Workout for the muscle of the arm and calves', 1);
+
```

Comment:
Since the user_id for the workout table can be nullable, I'd suggest that you leave it as null, it should only be filled when an actual user creates a workout routine.

Reply:
This is actually meant for my local database, not supposed to make it to github.